ui/app/registration/controllers/createPatientController.js

```diff
@@ -54,6 +58,21 @@ angular.module('bahmni.registration')
                         };
                     }).value();
                 };
+                
+                var setDefaultLocation = function (personAttributeType) {
+                    console.log("Default = ",defaults[personAttributeType.name]);
```

Comment:
@Arjun-Go is this console.log added intentionally or can be removed ?

Reply:
@binduak That shouldn't be there, will remove it.